src/App.tsx

```diff
@@ -5,13 +5,14 @@ import Home from "./components/home/Home";
 import Dashboard from "./components/dashboard/Dashboard";
 import Invoices from "./components/invoices/Invoices";
 import { PATHS } from "./urls/urls";
-import { Login } from "./components/login/Login";
+import { Register } from "./components/register/Register";
 import { useState } from "react";
+import { Login } from "./components/login/Login";
 
 function App() {
   //todo // state jest tylko po to aby przetestować ekran login
   const [state, setState] = useState(true);
```

Comment:
Rozumiem że ten boolean symuluje to czy ktoś jest zalogowany lub nie?

Reply:
To było tylko do testów, już się pozbyłem.